docs/articles/documentation/recipes/debugging-with-visual-studio-code.md

```diff
@@ -44,6 +45,7 @@ You will need to add the following configuration to the `launch.json` file.
 This configuration contains the following attributes:
 
 * `type` - specifies the type of the configuration. Set to `node` for a Node.js configuration.
+* `protocol` - specifies the Node.js [debugger wire protocol](https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_supported-nodelike-runtimes). Set to `inspector` to use the [Inspector protocol](https://chromedevtools.github.io/debugger-protocol-viewer/v8/). This protocol is available in Node.js v6.3 (or v6.9 for Windows) or newer. If you are using an older Node.js, do not specify the `protocol` option. In this instance, the `Legacy` protocol will be used, which may cause issues with source maps while debugging test files.
```

Comment:
Quite odd general phrasing. It sound like `issues with source maps while debugging test files` is some kind of feature and to enable it you need to avoid specifying `protocol`.